python/ray/remote_function.py

```diff
@@ -109,10 +109,10 @@ def _remote(self,
         worker = ray.worker.get_global_worker()
         worker.check_connected()
 
-        if self._last_export_session < worker._session_index:
-            # If this function was exported in a previous session, we need to
+        if self._last_export_session != worker.session_name:
```

Comment:
Hey, nice catch! I'm actually making some of these same fixes in https://github.com/ray-project/ray/pull/4844. There is a one-to-one correspondence between `session_name` and `driver_id`, right? I used `driver_id` in order to not introduce another concept.

Reply:
Oh interesting, actually `session_name` is for the cluster, and there could be many drivers sharing the same cluster, so they are not actually the same concept, right?

Reply:
Yes, a session could have multiple drivers. But I am not sure which way could be better. In theory, a remote function / actor should be unique in a session, and their lifetime is also based on the session, so we just need to make use of the session name.